vision/google/cloud/vision/_gax.py

```diff
@@ -66,7 +68,12 @@ def annotate(self, images=None, requests_pb=None):
             requests = requests_pb
 
         annotator_client = self._annotator_client
-        responses = annotator_client.batch_annotate_images(requests).responses
+        try:
+            api_result = annotator_client.batch_annotate_images(requests)
+            responses = api_result.responses
+        except RetryError as exception:
```

Comment:
So the `RetryError` is pretty [specific](https://github.com/googleapis/gax-python/blob/b938df8b343d793f524b22d4b0cbe86541ba42b0/google/gax/errors.py#L77), and I'm not 100% sure that this will work in a general sense.

Let me know if you have a strategy that would work.